src/RuleTicketCollection.php

```diff
@@ -141,7 +141,7 @@ public function prepareInputDataForProcess($input, $params)
                             $input[$input_key] = [];
                         }
 
-                        $input[$input_key][] = $actor['items_id'];
+                        $input[$input_key][$actor['items_id']] = $actor['items_id'];
```

Comment:
Unless I am wrong, this list may contain actors that are using numeric keys (actors set from API for instance). Using id as key may overwrite them.

```suggestion
                        if (in_array($actor['items_id'], $input[$input_key])) {
                            continue;
                        }

                        $input[$input_key][] = $actor['items_id'];
```